crc32c_config.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 The CRC32C Authors. All rights reserved.
```

Comment:
this file should not be here i think

Reply:
This file is auto generated by cmake (can't say I 100% know how), so bazel need to get this from somewhere.